package.json

```diff
@@ -105,7 +105,10 @@
       }
     },
     "rules": {
-      "arrow-parens": ["error", "always"],
+      "arrow-parens": [
+        "error",
+        "always"
+      ],
```

Comment:
IMHO it's good to have this whitespace change (even if you'd prefer to have it in other PR) because npm does this automatically and it's annoying to revert every time.